CONTRIBUTING.md

```diff
@@ -13,7 +13,11 @@ section.
 ## Modules
 
 * Prefer the use of the singular form in module name (or use "multi")
-* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module)
+* Use the [description template](https://github.com/OCA/maintainer-tools/tree/master/template/module) but remove sections with no meaningful content.
```

Comment:
Not everyone agrees with removing empty sections (although I do). They prefer to put texts like "Nothing to do" and so on. This is because they prefer consistency against shortness.